examples/basic-nextjs-pages-router/tsconfig.json

```diff
@@ -0,0 +1,36 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "components/*": ["src/components/*"],
+      "lib/*": ["src/lib/*"],
+      "temp/*": ["src/temp/*"],
+      "assets/*": ["src/assets/*"],
```

Comment:
".sitecore" alias is missing
https://github.com/Sitecore/content-sdk/blob/dev/packages/create-content-sdk-app/src/templates/nextjs-app-router%20(beta)/tsconfig.json#L17-L19